### PR TITLE
Modifying README.md for application acceptance process

### PR DESCRIPTION
**Pull request by SgStapleton**
Modifying README.md to include information on the project application acceptance process

### CODE REVIEW

**Review by rbeyer — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -99,13 +99,34 @@ their charters. The TSC cannot mandate existing working groups alter their
 charters. If such a situation arises, the TSC may decide to revoke the group's
 charter.
 
+## Project Application Acceptance Criteria
+
+All applications to the TSC must follow the application template. They are to be
```
Comment (jessemapel) [inline README.md]: A link to the application template would be good here

Comment (jessemapel) [inline README.md:104]: A link to the application template would be good here

File: README.md [hunk 2]
```diff
@@ -99,13 +99,34 @@ their charters. The TSC cannot mandate existing working groups alter their
...
   2. have two approvals
 
-Exceptions to the above are minor typo fixes or cosmetic changes that don't alter the meaning of a document.  Those edits can be made via a PR which only requires a single approval (not two, and not open for 24 h) to be merged.
+Exceptions to the above are minor typo fixes or cosmetic changes that don't
+alter the meaning of a document.  Those edits can be made via a PR which only
+requires a single approval (not two, and not open for 24 h) to be merged.
+
+Another exception to the above would be PRs related to the application
+acceptance process of new software projects into the TSC. This is covered in the
+"Project Application Acceptance Criteria" section.
```
Comment (jessemapel) [inline README.md:129]: I think this paragraph is clearer if you just say that applications follow a different process and then link to the section. See this SO question (https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown/7015050#7015050) for how to create portable links to sub-headings.

Comment (rbeyer) [reply README.md:129]: This PR now has conflicts with master.

Jesse, can you please determine whether this has been overcome by #66, and we can just close this PR, or refactor the intent of these changes on top of master?

Comment (jessemapel) [reply README.md:129]: Yep this is obsolete.

**Comment by jessemapel** [comment]
Obsoleted by #66

**Pull request closed without merging**